client/src/components/ProductItems/ProductItems.module.scss

```diff
@@ -0,0 +1,31 @@
+.ProductItems {
+	* {
+		color: black;
+		margin-left: 2%;
+		margin-right: 2%;
```

Comment:
Paddings, margins und höhen wenns geht nicht in sondern rem oder px.
Grund ist, dass Prozente hier total unvorhersehbar sind und nur chaos stiften.